pyccl/halos/profiles.py

```diff
@@ -1551,3 +1554,262 @@ def _Ns(self, M, a):
         M1 = 10.**(self.lM1_0 + self.lM1_p * (a - self.a_pivot))
         alpha = self.alpha_0 + self.alpha_p * (a - self.a_pivot)
         return np.heaviside(M-M0, 1) * (np.fabs(M-M0) / M1)**alpha
+
+
+class SatelliteShearHOD(HaloProfileHOD):
+    def __init__(self, c_M_relation, a1h=0.001, b=-2,
+                 lmax=6, integration_method='FFTLog',
+                 **kwargs):
+        '''
```

Comment:
The docstring describing the init method usually goes after the class declaration. Check how we do it here for instance https://github.com/LSSTDESC/CCL/blob/d5408a777ab5fc4b2ae6fada2114a168b9bacee4/pyccl/halos/profiles.py#L624